media-plugins/grilo-plugins/metadata.xml

```diff
@@ -11,8 +11,8 @@
     <flag name="flickr">Enable Flickr support</flag>
     <flag name="freebox">Build support for viewing TV channels using Freebox</flag>
     <flag name="gnome-online-accounts">Enable <pkg>net-libs/gnome-online-accounts</pkg> support</flag>
+    <flag name="localsearch">Build support for content discovery using the <pkg>app-misc/tinysparql</pkg> indexer</flag>
     <flag name="thetvdb">Build support for thetvdb.com</flag>
-    <flag name="tracker">Build support for content discovery using the <pkg>app-misc/tracker</pkg> indexer</flag>
```

Comment:
this is purely educational and no instruction to fix this PR, but: 0.3.16 is still around, so you can't remove this.